examples/tutorials/doc_examples/examples/cached_prop.py

```diff
@@ -8,18 +8,29 @@
 #
 # Thanks for using Enthought open source!
 
-# cached_prop.py - Example of @cached_property decorator
-# --[Imports]------------------------------------------------------------------
-from traits.api import HasPrivateTraits, List, Int, Property, cached_property
+"""
+This example demonstrates the use of Property, with caching and notifcation
+support.
 
+"""
 
-# --[Code]---------------------------------------------------------------------
-class TestScores(HasPrivateTraits):
+from traits.api import (
+    cached_property, HasPrivateTraits, Str, observe, Property,
+)
 
-    scores = List(Int)
```

Comment:
I rewrote the whole example to workaround the current awkwardness with comparison mode in `List`. See #1122.

Reply:
I don't know if that matters too much any more, but the rewritten example looks fine anyway.

Reply:
Indeed there is no need to workaround the comparison mode awkwardness because #1122 is now resolved.
If this rewritten example looks fine then I will keep this as it is now (so there is no need to review this again).